docs: docker cli (#133)
kubeshop/testkube-docs: docs/cli/testkube_upgrade.md

### Options

```

>       --agent-prefix string       usually don't need to be changed [required for custom cloud mode] (default "agent")

Reviewer: ```suggestion
      --agent-prefix string       Usually doesn't need to be changed [required for custom cloud mode] (default "agent")
```

      --agent-token string        Testkube Pro agent key [required for centralized mode]
      --agent-uri string          Testkube Pro agent URI [required for centralized mode]

>       --api-prefix string         usually don't need to be changed [required for custom cloud mode] (default "api")

Reviewer: ```suggestion
      --api-prefix string         Usually doesn't need to be changed [required for custom cloud mode] (default "api")
```

      --chart string              chart name (usually you don't need to change it) (default "kubeshop/testkube")
      --docker-container string   Docker container name for Testkube Docker Agent (default "testkube-agent")

>       --dry-run                   dry run mode - only print commands that would be executed

Reviewer: ```suggestion
      --dry-run                   Dry run mode - only print commands that would be executed
```

>       --embedded-nats             embedded NATS server in agent

Reviewer: ```suggestion
      --embedded-nats             Embedded NATS server in agent
```

      --env-id string             Testkube Pro environment id [required for centralized mode]
      --feature-logs-v2           Logs v2 feature flag

>   -h, --help                      help for upgrade

Reviewer: ```suggestion
  -h, --help                      Help for upgrade
```

>       --logs-prefix string        usually don't need to be changed [required for custom cloud mode] (default "logs")

Reviewer: ```suggestion
      --logs-prefix string        Usually doesn't need to be changed [required for custom cloud mode] (default "logs")
```

      --logs-uri string           Testkube Pro logs URI [required for centralized mode]

>       --master-insecure           should client connect in insecure mode (will use http instead of https)

Reviewer: ```suggestion
      --master-insecure           Should client connect in insecure mode (will use http instead of https)
```

      --name string               installation name (usually you don't need to change it) (default "testkube")

>       --namespace string          namespace where to install (default "testkube")

Reviewer: ```suggestion
      --namespace string          Namespace of the location to install (default "testkube")
```

      --no-confirm                don't ask for confirmation - unatended installation mode

>       --no-minio                  don't install MinIO

Reviewer: ```suggestion
      --no-minio                  Don't install MinIO
```
